src/main/resources/db/changelog/migration.xml

```diff
@@ -32,7 +32,7 @@
             <column name="phone_number" type="varchar(255)">
                 <constraints nullable="false"/>
             </column>
-            <column name="user_role" type="ENUM('USER','ADMIN','TEACHER')" defaultValue="USER">
+            <column name="role" type="ENUM('USER','ADMIN','TEACHER')" defaultValue="USER">
                 <constraints nullable="false"/>
             </column>
         </createTable>
```

Comment:
Foreign keys should be like 
 <addForeignKeyConstraint baseColumnNames="user_id"
                             baseTableName="user"
                             constraintName="fk_user_id"
                             referencedColumnNames="id"
                             referencedTableName="owner"/>